sound/soc/intel/boards/bxt_pcm512x.c

```diff
@@ -204,6 +204,7 @@ static struct snd_soc_dai_link dailink[] = {
 		.init = broxton_hdmi_init,
 		.dpcm_playback = 1,
 		.no_pcm = 1,
+		.trigger[0] = SND_SOC_DPCM_TRIGGER_POST,
```

Comment:
@libinyang If we are changing the behavior for the iDisp link, then you need to test all configurations with the SST/cAVS driver. we *cannot* use a solution for iDisp that works only for SOF. So please make sure this is tested on multiple platforms before we jump to this different trigger mode.

Reply:
specifically all machines drivers I looked at use this for iDisp:
.trigger = {
			SND_SOC_DPCM_TRIGGER_POST, SND_SOC_DPCM_TRIGGER_POST},

so why do we only set the first element?

Reply:
> specifically all machines drivers I looked at use this for iDisp:
> .trigger = {
> SND_SOC_DPCM_TRIGGER_POST, SND_SOC_DPCM_TRIGGER_POST},
> 
> so why do we only set the first element?

@plbossart 
I only set for the first element because there is no capture for HDMI audio.

Reply:
@plbossart Currently, I only change the setting of the platform bxt_pcm512x.c. The default mode is still the same as before. This means other platforms will not be impacted. And yes, I will test on other platforms to see if others need such setting.

> @libinyang If we are changing the behavior for the iDisp link, then you need to test all configurations with the SST/cAVS driver. we _cannot_ use a solution for iDisp that works only for SOF. So please make sure this is tested on multiple platforms before we jump to this different trigger mode.